public/js/test/mochitest/browser_dbg_interrupts.js

```diff
@@ -0,0 +1,89 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/**
+ * Test resuming from button and keyboard shortcuts.
+ */
+
+const TAB_URL = EXAMPLE_URL + "doc-script-switching-01.html";
+const sourceURLs = [
+  "code-script-switching-01.js",
+  "code-script-switching-02.js"
+];
+const resumeKey = "VK_F8";
+
+add_task(function*() {
+  const dbg = yield initDebugger(TAB_URL, ...sourceURLs);
+  const { selectors: { getBreakpoints, getBreakpoint }, getState } = dbg;
+  const sources = [];
+  for (let url of sourceURLs) {
+    sources.push(findSource(dbg, url));
+  }
```

Comment:
I think it's more descriptive and simple if you just do:

```js
const source1 = findSource(dbg, "switching-01.js");
const source2 = findSource(dbg, "switching-02.js");
```

Admittedly, it's not much more descriptive because the script names are just 01 and 02, but it's shorter code anyway. Most tests will only deal with 2 or 3 sources so it's not a big deal to do this for each, and it's easier to debug (you see `source1` and `source2` in scope instead of an array)